plugins/API/API.php

```diff
@@ -69,7 +69,7 @@ class API extends \Piwik\Plugin\API
      * For Testing purpose only
      * @var int
      */
-    public static $_autoSuggestLookBack = 60;
```

Comment:
This one is used in various plugins, so we actually can only change that in a major release.

Reply:
Reverted the underscore rule.